index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LizanshPortfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+            <div class="header">
+                <img class="img1" src="./img/image_geometry_1.png" alt="">
+                <img class="img2" src="./img/ic_menu.png" alt="">
+            </div>
+    </header>
+
+    <div class="name">
+            <h1>Lizansh Mathur</h1>
+            <br>
+
+            <p>
+                Hello! I am a software developer! I can 
+                help you build a product, feature or website.
+                 Take a look at my work.
+                <br> 
+                <br>
+                If you like what you see and have a project you need
+                 coded, don’t hesitate to contact me.
+             </p>
+                <br>
+                
+                <div class="button">
+                    <button class="rectangle">Start collaboration</button>
+                </div>
+    </div>
```

Comment:
- [ ] Your headline section should be `section` tag, It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info